### PR TITLE
reSTify PEP 343

### PR DESCRIPTION
**Pull request by mozillazg**
refer #4

### CODE REVIEW

File: pep-0343.txt [hunk 1]
```diff
@@ -5,810 +5,825 @@ Last-Modified: $Date$
...
+   purpose, since it's not an exception that I want people to catch:
+   I want it to turn into a traceback which is seen by the
+   programmer who then fixes the code.  So now I believe they
+   should both raise ``RuntimeError``.
+   There are some precedents for that: it's raised by the core
+   Python code in situations where endless recursion is detected,
+   and for uninitialized objects (and for a variety of
+   miscellaneous conditions)."
+
+2. It is fine to raise AttributeError instead of ``TypeError`` if the
```
Comment (brettcannon) [inline pep-0343.txt]: Missed putting ``AttributeError`` in surrounding "``".

File: pep-0343.txt [hunk 2]
```diff
@@ -5,810 +5,825 @@ Last-Modified: $Date$
...
-       (This example is easily modified to implement the other
-       relatively stateless examples; it shows that it is easy to avoid
-       the need for a generator if no special state needs to be
-       preserved.)
+   This isn't thread-safe, of course, but neither is doing this
+   same dance manually.  In single-threaded programs (for example,
+   in scripts) it is a popular way of doing things.
 
-    5. Redirect stdout temporarily:
+6. A variant on opened() that also returns an error condition::
```
Comment (brettcannon) [inline pep-0343.txt]: ``opened()``

File: pep-0343.txt (hunk 1 above)
Comment (mozillazg) [inline pep-0343.txt]: @brettcannon Done.

File: pep-0343.txt (hunk 2 above)
Comment (mozillazg) [inline pep-0343.txt]: @brettcannon Done.

**Comment by mozillazg** [comment]
@brettcannon Thanks for your review! Please check again.

**Comment by brettcannon** [comment]
Thanks!

**Pull request merged**